pootle/apps/pootle_project/models.py

```diff
@@ -286,14 +286,14 @@ def is_terminology(self):
 
     @property
     def vfolders(self):
```

Comment:
@unho, this property should also check if `virtualfolder` app is used. It was added before this PR changes so I don't know if we can do it here.